supyr_struct/buffer.py

```diff
@@ -242,13 +290,11 @@ class BytearrayBuffer(bytearray, Buffer):
 
     Uses os.SEEK_SET, os.SEEK_CUR, and os.SEEK_END when calling seek.
     '''
-    __slots__ = ('_pos')
+    __slots__ = ('_pos',)
```

Comment:
Nice catch. I don't know how it was working before.

Reply:
It catches it specifically as something that is wrong, but accepted